component/admin/fields/jeveventcalendar.php

```diff
@@ -29,7 +29,9 @@ protected function getInput()
 		$clist= $this->form->jevdata[$this->name]["clist"];
 		$nativeCals = $this->form->jevdata[$this->name]["nativeCals"];
 		
-		if ($native && $clistChoice)
+		$params = JComponentHelper::getParams(JEV_COM_COMPONENT);
+
+		if ($native && $clistChoice && (int) $params->get('defaultcat', 1) === 1)
 		{
```

Comment:
This condition means that the function preselectCategory is not defined and it needs a definition for all situations where there is more than one nativeCal available.  

It probably needs to set the category id to zero when defaultcat is zero